### PR TITLE
[ticket/16010] Enforce alphabetical order for events in events.md

### PR DESCRIPTION
**Pull request by marc1706**
PHPBB3-16010

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16010

### CODE REVIEW

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -28,29 +28,29 @@ acp_bbcodes_edit_fieldsets_after
 * Since: 3.1.0-a3
 * Purpose: Add settings to BBCode add/edit form
 
-acp_email_group_options_append
+acp_email_find_username_append
 ===
 * Location: adm/style/acp_email.html
 * Since: 3.1.7-RC1
-* Purpose: Add content at the end of the group options select box 
+* Purpose: Add content at the end of the fimd username link
```
Comment (mrgoldy) [inline phpBB/docs/events.md]: Find

File: phpBB/docs/events.md [hunk 2]
```diff
@@ -28,29 +28,29 @@ acp_bbcodes_edit_fieldsets_after
...
 
-acp_email_group_options_prepend
+acp_email_find_username_prepend
 ===
 * Location: adm/style/acp_email.html
 * Since: 3.1.7-RC1
-* Purpose: Add content at the start of the group options select box 
+* Purpose: Add content at the start of the fimd username link
```
Comment (mrgoldy) [inline phpBB/docs/events.md]: Find

File: phpBB/docs/events.md [hunk 3]
```diff
@@ -281,29 +281,29 @@ acp_overall_header_stylesheets_after
...
-* Purpose: Add content before the sourse forum select form
+* Purpose: Add content before the destiny forum select form
 
-acp_permission_forum_copy_dest_forum_append
+acp_permission_forum_copy_src_forum_append
 ===
 * Location: adm/style/permission_forum_copy.html
 * Since: 3.1.7-RC1
-* Purpose: Add content after the destiny forum select form
+* Purpose: Add content after the sourse forum select form
```
Comment (mrgoldy) [inline phpBB/docs/events.md]: Source

File: phpBB/docs/events.md [hunk 4]
```diff
@@ -281,29 +281,29 @@ acp_overall_header_stylesheets_after
...
 
-acp_permission_forum_copy_dest_forum_prepend
+acp_permission_forum_copy_src_forum_prepend
 ===
 * Location: adm/style/permission_forum_copy.html
 * Since: 3.1.7-RC1
-* Purpose: Add content before the destiny forum select form
+* Purpose: Add content before the sourse forum select form
```
Comment (mrgoldy) [inline phpBB/docs/events.md]: Source

File: phpBB/docs/events.md [hunk 5]
```diff
@@ -281,29 +281,29 @@ acp_overall_header_stylesheets_after
 * Purpose: Add assets after stylesheets within the `<head>` tags in the ACP.
 Note that INCLUDECSS will not work with this event.
 
-acp_permission_forum_copy_src_forum_append
+acp_permission_forum_copy_dest_forum_append
 ===
 * Location: adm/style/permission_forum_copy.html
 * Since: 3.1.7-RC1
-* Purpose: Add content after the sourse forum select form
+* Purpose: Add content after the destiny forum select form
```
Comment (3D-I) [inline phpBB/docs/events.md]: destination

File: phpBB/docs/events.md [hunk 6]
```diff
@@ -281,29 +281,29 @@ acp_overall_header_stylesheets_after
...
 
-acp_permission_forum_copy_src_forum_prepend
+acp_permission_forum_copy_dest_forum_prepend
 ===
 * Location: adm/style/permission_forum_copy.html
 * Since: 3.1.7-RC1
-* Purpose: Add content before the sourse forum select form
+* Purpose: Add content before the destiny forum select form
```
Comment (3D-I) [inline phpBB/docs/events.md]: destination

File: phpBB/docs/events.md (hunk 5 above)
Comment (marc1706) [reply phpBB/docs/events.md]: Heh, yeah. Not sure how that happened :)

**Comment by 3D-I** [comment]
Lol, my pleasure.

**Pull request merged**